src/admin.rs

```diff
@@ -186,10 +187,7 @@ async fn handle_dashboard(_req: Request<Incoming>) -> Response<Full<Bytes>> {
 fn x509_to_pem(x509: &X509) -> String {
     match x509.to_pem() {
         Err(e) => format!("<pem construction error: {e}>"),
-        Ok(vec) => match String::from_utf8(vec) {
-            Err(e) => format!("<utf8 decode error: {e}>"),
-            Ok(s) => s,
-        },
+        Ok(vec) => base64::encode_block(&vec),
```

Comment:
Envoy also encodes certs in base64 so this seams good.

Should the field name be changed to reflect that it's not a plaintext PEM anymore?

Reply:
inline_bytes I don't love either so I didn't want to specifically call out that name 😂 field makes sense including inline vs being a reference to a file though so maybe it's not all bad.

Reply:
No need to change that, since base64 is also intuitive